packages/mask/src/plugins/Game/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { Constant } from '@masknet/web3-shared-base'
+
+export interface GameDialogEvent {
+    open: boolean
+    tokenProps?: GameNFT
+}
+
+export interface GameRSSNode {
+    address: string
+    signature: string
+    games: Record<string, Constant>
+}
+
+export interface GameInfo {
+    id: number
+    name: string
+    image: string
+    description: string
+    twitterId: string
+    url: string
+    wallet: boolean
+    rank: number
+    width: number
+    height: number
+}
+
+export interface GameNFT {
+    tokenId?: string
+    contract?: string
+    chainId?: number
```

Comment:
Please import it from `@masknet/web3-shared-evm`.

```suggestion
    chainId?: ChainId
```